spec/factories/users.rb

```diff
@@ -24,9 +24,10 @@
 #  index_users_on_reset_password_token  (reset_password_token) UNIQUE
 #  index_users_on_username              (username) UNIQUE
 #
+
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.